### PR TITLE
Create cuda.vim

### PR DESCRIPTION
**Pull request by sunbubble**
Added maker for cuda: nvcc.

### CODE REVIEW

File: autoload/neomake/makers/ft/cuda.vim [hunk 1]
```diff
@@ -0,0 +1,12 @@
+" vim: ts=4 sw=4 et
+
+function! neomake#makers#ft#cuda#EnabledMakers()
+    return ['nvcc']
+endfunction
+
+function! neomake#makers#ft#cuda#nvcc()
+    return {
+        \ 'exe': 'nvcc',
+        \ 'errorformat': '%f\(%l\):%m',
```
Comment (blueyed) [inline autoload/neomake/makers/ft/cuda.vim]: There will be no error type assigned (error, warning, info etc).
Could this information be parsed from the output, e.g. with some special argument/format?

**Comment by blueyed** [comment]
Please also add it to the list of makers in the README.

File: autoload/neomake/makers/ft/cuda.vim (hunk 1 above)
Comment (sunbubble) [inline autoload/neomake/makers/ft/cuda.vim]: I see what you mean, I'll refine it.

**Comment by sunbubble** [comment]
Done.

**Comment by blueyed** [comment]
Thanks!

**Pull request merged**